ui/mainwindow.ui

```diff
@@ -89,7 +91,7 @@ background-position:bottom right;
      <x>0</x>
      <y>0</y>
      <width>829</width>
-     <height>27</height>
+     <height>39</height>
```

Comment:
I used the designer to change the stylesheet.
Was wondering myself about this.
So what I did was resetting the file and redo the changes to the stylesheet.
(I carefully looked not to accidentally move something in the designer) 
But again, this change co-ocurred after changing the style, so I thought if the designer thinks this is relevant to the change, I just commit that too.